engine/src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala

```diff
@@ -95,6 +95,26 @@ trait WdlStandardLibraryImpl extends WdlStandardLibraryFunctions {
     */
   override def read_float(params: Seq[Try[WdlValue]]): Try[WdlFloat] = read_string(params) map { s => WdlFloat(s.value.trim.toDouble) }
 
+  override def sub(params: Seq[Try[WdlValue]]): Try[WdlString] = {
+    def extractArguments = params.size match {
+      case 3 => Success((params.head, params(1), params(2)))
+      case n => Failure(new IllegalArgumentException(s"Invalid number of parameters for engine function sub: $n. sub takes exactly 3 parameters."))
+    }
+
+    def validateArguments(values: (Try[WdlValue], Try[WdlValue], Try[WdlValue])) = values match {
+      case (Success(strValue), Success(WdlString(pattern)), Success(replaceValue))
```

Comment:
Yep. I guess it's a matter of definition, but if we do want to have regex capabilities then I don't see any way around that except having a different function that consider `pattern` as a simple string.